src/controller/order-controller.ts

```diff
@@ -127,3 +162,70 @@ export async function captureOrderController(fastify: FastifyInstance) {
     },
   });
 }
+
+// Patch order
+export type ShippingOption = {
+  id: string;
+  label: string;
+  type: string;
+  selected: boolean;
+  amount: {
+    value: string;
+    currency_code: string;
+  };
+};
+
+async function patchOrderHandler(request: FastifyRequest, reply: FastifyReply) {
+  const { selectedShippingOption, orderID } = request.body as {
+    selectedShippingOption: ShippingOption;
+    orderID: string;
+  };
+  const { access_token: accessToken } = await getAuthToken();
+
+  let shippingPayload = {
+    selectedShippingOption,
+    orderID,
+  };
+
+  const data = await onShippingChange(accessToken, shippingPayload);
+  // Send a response with a 204 status code and no body
+  reply.code(204).send();
```

Comment:
Does the `patch` call always return a 204? Or is there a chance it returns a 200? Wondering if we need to check the `data` variable above to see if there's a response or not.

```js
if (data) {
  reply.send(data);
} else {
  reply.code(204).send();
}
```